playbooks/ps_80.yml

```diff
@@ -100,13 +103,25 @@
 
   - name: install percona-mysql-shell package
     include_tasks: ../tasks/install_pshell.yml
-
-  - name: check that Percona Server version is correct
-    shell: /package-testing/version_check.sh ps80 &> version-check.log
-
-  - name: Log check for warnings in the previous stage
-    command: /package-testing/scripts/log-warning-check.sh version-check.log
-    when: (lookup('env', 'check_warnings') == "yes")
+    when: (lookup('env', 'install_mysql_shell') == "yes") 
+
+  - name: Handle check that Percona Server version is correct
+    block: 
+      - name: Run the version_check.sh script
+        shell: /package-testing/version_check.sh ps80 &> version-check.log
+
+      - name: Log check for warnings in the previous stage
+        command: /package-testing/scripts/log-warning-check.sh version-check.log
+        when: (lookup('env', 'check_warnings') == "yes")
+
+    rescue:
+      - name: Print the output of the version_check.sh logfile on the script failure
+        command: cat "version-check.log"
+        register: versioncheckoutput
```

Comment:
Yes regarding that part, if we do cat command in the register, It shows the stdout and stderror.
I tested it with debug and we got 2 outputs so I removed the debug step so that we get 1 output.